ui/src/ifql/components/FuncArg.tsx

```diff
@@ -84,6 +105,10 @@ class FuncArg extends PureComponent<Props> {
     }
   }
 
+  private get value(): string {
+    return `${this.props.value}`
```

Comment:
I think that toString is less expensive than interpolation.

Reply:
done and done